app/src/main/java/org/simple/clinic/di/AppComponent.kt

```diff
@@ -3,14 +3,13 @@ package org.simple.clinic.di
 import dagger.Component
 import org.simple.clinic.ClinicApp
 import org.simple.clinic.ReleaseClinicApp
-import org.simple.clinic.activity.TheActivityComponent
+import org.simple.clinic.main.TheActivityComponent
 import org.simple.clinic.facility.FacilitySync
 import org.simple.clinic.login.OtpSmsReceiver
 import org.simple.clinic.newentry.clearbutton.ClearFieldImageButton
 import org.simple.clinic.storage.Migration_27_28
 import org.simple.clinic.storage.Migration_29_30
 import org.simple.clinic.storage.Migration_34_35
-import org.simple.clinic.storage.Migration_48_49
```

Comment:
Curious, what happened to this migration?

Reply:
This was an unused import. I think a larger migration was broken into two, and the injection moved to the later commit, but the original import was left.